data/nitesh.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "B Niesh",
+  "bio": "Web Developer | Open Source Contributor | UI/UX  | ",
+  "tags": ["C++", "JavaScript", "Open Source", "DevOps", "Docker", "python" , "Aws", "React", "PHP", "SQL" ],
+  "socials": [
+    
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/Anushka-codergirl"
+    },
+    {
+      "icon": "FaLink",
+      "url": "https://anushkaraj.me"
+    }
+  ],
+  "links": [
+    {
+      "name": "Follow my Open Source on GitHub",
+      "url": "https://github.com/Moon1knight",
+      "icon": "FaGithub"
+    },
```

Please remove this trailing comma here! 

```suggestion
    }
```